public/app/features/explore/state/reducers.ts

```diff
@@ -624,7 +623,12 @@ export const processQueryResponse = (
   const logsResult = processor.getLogsResult();
 
   // For Angular editors
-  state.eventBridge.emit('data-received', legacy);
+  if (true) {
```

Comment:
Is there a good way to check if the datasource has an angular editor?

Reply:
yes, if datasourceInstance.components.QueryCtrl

Reply:
thanks